roles/disaster_recovery/examples/dr_ovirt_setup.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Setup oVirt environment
+  hosts: localhost
+  connection: local
+  vars_files:
+     - ovirt_passwords.yml
+     - disaster_recovery_vars.yml
+  roles:
+     - disaster_recovery
```

Comment:
missing collection usage( collections: ovirt.ovirt )

Reply:
Done.
Commit "updating the role name" was updated with the fix.